components/http_client.rst

```diff
@@ -130,14 +130,18 @@ each request (which overrides any global authentication)::
 
     // Use the same authentication for all requests
     $httpClient = HttpClient::create([
-        // HTTP Basic authentication with only the username and not a password
+        // HTTP Basic authentication (there are multiple ways of configuring it)
```

Comment:
I'm not sure it's useful to document all the alternatives, do you?

Reply:
In other circunstantes I'd agree, but considering that all the alternatives can be shown in just 3 lines, I'd say it's worth it in this case.